src/pages/Admin/AdminMainLayout.tsx

```diff
@@ -35,14 +36,17 @@ export const AdminMainLayout = () => {
     fetchAdminCohorts();
   }, [fetchAdminCohorts]);
 
-  return cohortsList?.length ? (
+  // return cohortsList?.length ? (
+  return (
     <div>
       <Container>
         <AdminNavbar />
+        <BreadcrumbMenu />
         <Outlet context={{ cohorts: cohortsList }} />
       </Container>
     </div>
-  ) : null;
+  );
+  // ) : null;
```

Comment:
this is weird, you shouldn't be adding these changes here since i've added it in main already. If you do `rebase`, make sure you don't do `merge` cause they will conflict